src/main/resources/application.yml

```diff
@@ -30,6 +31,7 @@ spring:
 
 server:
   port: 9002
+  requestLogginFilterEnabled: ${REQUEST_LOGGING_FILTER_ENABLED:false}
```

Comment:
We need to remember to have this configuration set to false, so we will not log every request.

Reply:
Yeah definitely one for debugging only.